config/conference.ts

```diff
@@ -25,7 +25,9 @@ const presentationSubmissionsOpenFrom = moment.parseZone('2019-04-30T08:00:00+08
 const presentationSubmissionsOpenUntil = moment.parseZone('2019-06-02T23:59:59+08:00')
 const votingOpenFrom = moment.parseZone('2019-06-08T17:00:00+08:00')
 const votingOpenUntil = moment.parseZone('2019-06-16T23:59:59+08:00')
```

Comment:
Should we change these dates? I Assume that we only know the conference date. What should we do about the rest of the dates? not sure

Reply:
We aren't locking them in until March.

Reply:
Unfortunately the only date we have programmed to hide is the conference date. Would have been a bit of extra work to get this. Perhaps a good post conference item to improve the site.